README.md

```diff
@@ -82,9 +82,10 @@ recommended in the warning message:
 conan profile update settings.compiler.libcxx=libstdc++11 tket
 ```
 
-Add the `tket.libs` repository to your remotes:
+Set the `tket-libs` repository as your remote:
 
 ```shell
+conan remote clean
```

Comment:
Do we want to add more explanations on this here in the readme why we are doing this? Is this something we are recommending for all users? Does this have negative side effects when working on other project as well?

Reply:
This is recommended when building tket, yes. It might affect other projects -- but then, so would the existing recommendation to add the tket-libs remote. I'll add a warning about this.

Reply:
Done.

Reply:
That is what I was assuming, thank you for adding the note.